src/borg/repository.py

```diff
@@ -959,6 +959,14 @@ def replay_segments(self, index_transaction_id, segments_transaction_id):
             self.exclusive = remember_exclusive
             self.rollback()
 
+    def get_storage_quota_use(self):
+        storage_quota_use = 0
+        for i, (segment, filename) in enumerate(self.io.segment_iterator()):
+            objects = list(self.io.iter_objects(segment))
+            for tag, key, offset, size, _ in objects:
+                storage_quota_use += header_size(tag) + size
+        return storage_quota_use
```

Comment:
a borg repo already maintains quota set and quota used, please just search in repository.py.

Reply:
I know, but when running rinfo, storage_quota_use is not loaded from disk, so I still need a function.
I'm gonna change the implementation of `get_storage_quota_use` to read from the hints file.